src/test/koa-npm-resolution.test.ts

```diff
@@ -0,0 +1,83 @@
+import {resolve as resolvePath} from 'path';
+import request from 'supertest';
+import test from 'tape';
+
+import createMiddleware from '../koa-npm-resolution';
+
+import {createAndServe, squeezeHTML} from './test-utils';
+
+test('transforms resolvable specifier in JavaScript module', async (t) => {
+  t.plan(1);
```

Comment:
what's this?

Reply:
The `tape` testing library uses this to express how many assertions should be encountered in the test, which is how it deals with ensuring that tests that are asynchronous, iterative or generative are actually testing all-the-things.  My tests here all have a single assertion, so these `t.plan(1)` calls look extraneous, but I don't mind them.  Tape supports using `t.end()` as an alternative, but I like the explicitness of the assertion count.

Reply:
https://github.com/substack/tape